R/cv_estimate_rt.R

```diff
@@ -69,8 +69,8 @@ cv_estimate_rt <- function(
     mse = function(y, m) (y - m)^2,
     mae = function(y, m) abs(y - m),
     deviance = function(y, m) {
-      devr <- y * log(m) - m
-      devy <- y * log(y) - y
+      devr <- m * log(m) - m
+      devy <- m * log(y) - y
```

Comment:
Actually, we should just reverse `(devr - devy)`. (This is on the `delay-hotfix` branch).